docs/tutorials/uky_flowsheet-solution.ipynb

```diff
@@ -1973,10 +1856,1523 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
    "id": "98846a70-f021-436c-ade6-20141d6f6f5c",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
I've made `tee=False` for the initialization solves, so that has shortened the output by a great deal. As for the the intermediates solves not converging, that's a known issue and is also present in the UKy flowsheet itself.